templates/_base/base.html

```diff
@@ -63,37 +63,40 @@
         </span>
         <ul class="p-navigation__links" role="menu">
           <li class="p-navigation__link" role="menuitem">
-            <a href="/openstack">OpenStack</a>
+            <a href="/openstack" {% if level_1 == 'openstack' %}class="active"{% endif %}>OpenStack</a>
           </li>
           <li class="p-navigation__link" role="menuitem">
-            <a href="/kubernetes">Kubernetes</a>
+            <a href="/kubernetes" {% if level_1 == 'kubernetes' %}class="active"{% endif %}>Kubernetes</a>
           </li>
           <li class="p-navigation__link" role="menuitem">
-            <a href="/ai-ml">人工知能/機械学習</a>
+            <a href="/ai-ml" {% if level_1 == 'ai-ml' %}class="active"{% endif %}>人工知能/機械学習</a>
           </li>
           <li class="p-navigation__link" role="menuitem">
-            <a href="/iot">デバイス</a>
+            <a href="/iot" {% if level_1 == 'iot' %}class="active"{% endif %}>デバイス</a>
           </li>
           <li class="p-navigation__link" role="menuitem">
-            <a href="/enterprise-support">商用サポート</a>
+            <a href="/enterprise-support" {% if level_1 == 'enterprise-support' %}class="active"{% endif %}>商用サポート</a>
           </li>
+          {% comment %}
           <li class="p-navigation__link" role="menuitem">
-            <a href="/blog">ブログ</a>
+            <a href="/blog" {% if level_1 == 'blog' %}class="active"{% endif %}>ブログ</a>
           </li>
+          {% endcomment %}
           <li class="p-navigation__link" role="menuitem">
-            <a href="/download">ダウンロード</a>
+            <a href="/download" {% if level_1 == 'download' %}class="active"{% endif %}>ダウンロード</a>
           </li>
         </ul>
       </nav>
     </div>
   </header>
 
   {% block header %}{% endblock %}
-
+<div id="main-content" class="u-no-margin--top">
```

Comment:
All the pages have main-content div in them so you'll either need to remove them or this div. Also, no margin top should not be needed.

Reply:
@anthonydillon 	I remove the main-content div from all the main pages... I have rebased now.